vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.Editor.Hints
+
+open Microsoft.VisualStudio.FSharp.Editor
+open FSharp.Compiler.CodeAnalysis
+open FSharp.Compiler.Symbols
+open FSharp.Compiler.Text
+open Hints
+
+module InlineParameterNameHints =
+
+    let private getHint (range: range, parameter: FSharpParameter) =
+        {
+            Kind = HintKind.ParameterNameHint
+            Range = range.StartRange
+            Parts = [ TaggedText(TextTag.Text, $"{parameter.DisplayName} = ") ]
+        }
+
+    let private doesParameterNameExist (parameter: FSharpParameter) = 
+        parameter.DisplayName <> ""
+
+    let isValidForHint (symbol: FSharpMemberOrFunctionOrValue) =
+        // is there a better way?
+        let isNotAnOperator = 
+            symbol.DeclaringEntity 
+            |> Option.exists (fun entity -> entity.CompiledName <> "Operators")
```

Comment:
This will only cover built-in operators, which is a minority for many use-cases.
For one, `Symbol` has `IsOperatorOrActivePattern` property, and then you can see if the `DisplayName` starts with `(` to distinguish one from another.

Reply:
Or even extend the type:
```fsharp
type FSharpMemberOrFunctionOrValue with
    member x.IsOperatorOrActivePattern =
      x.CompiledName.StartsWith "op_"
      || let name = x.DisplayName in

         if name.StartsWith "( " && name.EndsWith " )" && name.Length > 4 then
           name.Substring(2, name.Length - 4) |> String.forall (fun c -> c <> ' ')
         else
           false
```

Reply:
We also have `PrettyNaming.IsOperatorDisplayName`, not sure it should be used here.

Reply:
So, this method doesn't catch custom ops indeed, BUT customer operators case is filtered out elsewhere. This returns None for them so hints are not generated in the end. 
```
let ranges = parseResults.GetAllArgumentsForFunctionApplicationAtPosition symbolUse.Range.Start
```

I'm figuring out what's going on here, for now just saying that it works as expected (?) and maybe by coincidence :D

> We also have PrettyNaming.IsOperatorDisplayName, not sure it should be used here.

This I actually tried first but it doesn't detect built-in ops.